modules/exploits/linux/misc/redis_unauth_upload_file.rb

```diff
@@ -0,0 +1,126 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = GoodRanking
+
+  include Msf::Exploit::Remote::Tcp
```

Comment:
This would be great with Scanner.  I have a number of suggestions -- I'll shoot you over a PR shortly.